lib/parse.js

```diff
@@ -5,7 +5,7 @@ const languages = require('./languages');
 
 const constants = {
   ESCAPED_CHAR_REGEX: /^\\./,
-  QUOTED_STRING_REGEX: /^(['"`])((?:\\.|[^\1])+?)(\1)/,
+  QUOTED_STRING_REGEX:/^(['"`])((?:|[^\1])+?)(\1)/,
```

Comment:
Space after `QUOTED_STRING_REGEX:`

And I think the `+` should be inside, after the closing angle bracket `((?:[^\1]+)?)` and remove the `|`